### ISSUE TITLE
Rework on the avatar system

### ISSUE THREAD

**Issue by PadowYT2** [VertisanPRO/pterodactyl#251]
### Product

WemX Pro

### Describe the solution you'd like

The avatar system is currently not that good. By default, it is either the gravatar image from Imgur and not actual Gravatar or first letters of first+last name.

We should make it like gravatar does, get the avatar name by their email and encrypt it with md5 and it should show the same in admin and client pages

**Comment by VanillaChan6571** [comment]
I agree, admins/owners should have control over the avatar system instead of a third party. This will also reduce cross-site content if the avatar system is built in house (aka on wemx itself). This should also give owners/admins to set a default image when new users are created.
